### ISSUE TITLE
Store IERS database for local processing

### ISSUE THREAD

**Issue by piyanatk** [RadioAstronomySoftwareGroup/pyuvdata#715]
When reading in a file, `read_uvh5` uses  `astropy.utils.iers` to download the most recent IERS database to determine the accurate time and earth reference positions. The database, hosted at USNO, is often offline. Allowing a locally stored database file to be used when the server is down would be very helpful in analyses.

**Comment by bhazelton** [comment]
`read_uvh5` does not explicitly download the most recent IERS table, but it does do a calculation of LST that requires astropy to have a recent enough IERS table. Under the hood, `astropy` has a local stored IERS file that it uses if it is recent enough. If it is too old, it tries to download one from an external URL.

In older versions of `astropy`, the default location to download from was a USNO site, but the USNO sites have been taken down for ~6months for maintenance, so `astropy` recently released new versions that use other sites that are online. I'd suggest making sure you have the most recent version of astropy (3.2.3) to ensure that the IERS URL is up to date.

**Comment by piyanatk** [comment]
I upgrade to `astropy 3.2.3` and now have a timeout error. The HTTPS server is accessible from my network (but not the FTP server), so I am not sure what is causing the timeout.
```python
WARNING: failed to download https://datacenter.iers.org/data/9/finals2000A.all and ftp://cddis.gsfc.nasa.gov/pub/products/iers/finals2000A.all, using local IERS-B: HTTP Error 403: Forbidden;<urlopen error ftp error: timeout('timed out')> [astropy.utils.iers.iers]
```

**Comment by mkolopanis** [comment]
Last I heard the HTTPS server has a header issue right now, so while live, and downloadable astropy can't open it. The issue is fixed on master astropy/astropy#9499 but I do not believe there is a new version yet. The time out is happening because you are defaulting back to the ftp mirror. You are welcome to build astropy from source and stay up to date or ask your IT to make a rule to allow ftp traffic to the mirror address.

**Comment by piyanatk** [comment]
Got it. Thank you everyone.
